k9mail/src/main/java/com/fsck/k9/notification/NotificationIds.java

```diff
@@ -54,4 +59,14 @@ public static int getAuthenticationErrorNotificationId(Account account, boolean
     private static int getBaseNotificationId(Account account) {
         return account.getAccountNumber() * NUMBER_OF_NOTIFICATIONS_PER_ACCOUNT;
     }
+
+    public static int getNewSnoozedMessageId(Account account, int index) {
+//        if (index < 0 || index >= NUMBER_OF_SNOOZED_NOTIFICATIONS) {
+//            throw new IndexOutOfBoundsException("Invalid value: " + index);
+//        }
+        // just recycle/overwrite if they have too many snoozes
```

Comment:
We will just replace one of the snoozed notifications with another one. Hopefully the user won't have more than 10 snoozed notifications that they haven't dealt with yet; in that case we'd probably want to stop adding more instead of bombarding them, right?